chain/network/src/time.rs

```diff
@@ -0,0 +1,99 @@
+#![allow(dead_code)]
```

Comment:
Module-wide allows are pretty horrible. Strong feeling that this needs to be removed :) 

Perhaps you want just `pub mod time`? I'd imagine these API will be used across crates, not only in network.

Reply:
yes, that dead_code annotation is obsolete. done